main.tf

```diff
@@ -0,0 +1,141 @@
+  tags = merge({
+    Name        = "${local.resource_prefix.value}-customer-master"
+    Environment = local.resource_prefix.value
+    }, {
+    git_commit           = "d68d2897add9bc2203a5ed0632a5cdd8ff8cefb0"
+    git_file             = "terraform/aws/s3.tf"
+    git_last_modified_at = "2020-06-16 14:46:24"
+    git_last_modified_by = "nimrodkor@gmail.com"
+    git_modifiers        = "nimrodkor"
+    git_org              = "bridgecrewio"
+    git_repo             = "terragoat"
+    yor_trace            = "a7f01cc7-63c2-41a8-8555-6665e5e39a64"
+  })
+}
+
+resource "aws_s3_bucket" "financials" {
+  # bucket is not encrypted
+  # bucket does not have access logs
+  # bucket does not have versioning
+  bucket        = "${local.resource_prefix.value}-financials"
+  acl           = "private"
+  force_destroy = true
+  tags = merge({
+    Name        = "${local.resource_prefix.value}-financials"
+    Environment = local.resource_prefix.value
+    }, {
+    git_commit           = "d68d2897add9bc2203a5ed0632a5cdd8ff8cefb0"
+    git_file             = "terraform/aws/s3.tf"
+    git_last_modified_at = "2020-06-16 14:46:24"
+    git_last_modified_by = "nimrodkor@gmail.com"
+    git_modifiers        = "nimrodkor"
+    git_org              = "bridgecrewio"
+    git_repo             = "terragoat"
+    yor_trace            = "0e012640-b597-4e5d-9378-d4b584aea913"
+  })
+
+}
+
+resource "aws_s3_bucket" "operations" {
+  # bucket is not encrypted
+  # bucket does not have access logs
+  bucket = "${local.resource_prefix.value}-operations"
+  acl    = "private"
+  versioning {
+    enabled = true
+  }
+  force_destroy = true
+  tags = merge({
+    Name        = "${local.resource_prefix.value}-operations"
+    Environment = local.resource_prefix.value
+    }, {
+    git_commit           = "d68d2897add9bc2203a5ed0632a5cdd8ff8cefb0"
+    git_file             = "terraform/aws/s3.tf"
+    git_last_modified_at = "2020-06-16 14:46:24"
+    git_last_modified_by = "nimrodkor@gmail.com"
+    git_modifiers        = "nimrodkor"
+    git_org              = "bridgecrewio"
+    git_repo             = "terragoat"
+    yor_trace            = "29efcf7b-22a8-4bd6-8e14-1f55b3a2d743"
+  })
+}
+
+resource "aws_s3_bucket" "data_science" {
+  # bucket is not encrypted
+  bucket = "${local.resource_prefix.value}-data-science"
+  acl    = "private"
+  versioning {
+    enabled = true
+  }
+  logging {
+    target_bucket = "${aws_s3_bucket.logs.id}"
+    target_prefix = "log/"
+  }
+  force_destroy = true
+  tags = {
+    git_commit           = "d68d2897add9bc2203a5ed0632a5cdd8ff8cefb0"
+    git_file             = "terraform/aws/s3.tf"
+    git_last_modified_at = "2020-06-16 14:46:24"
+    git_last_modified_by = "nimrodkor@gmail.com"
+    git_modifiers        = "nimrodkor"
+    git_org              = "bridgecrewio"
+    git_repo             = "terragoat"
+    yor_trace            = "9a7c8788-5655-4708-bbc3-64ead9847f64"
+  }
+}
+
+resource "aws_s3_bucket" "logs" {
```

Comment:
<details><summary>
<img src="https://bc-network-graph-icons.s3.us-west-2.amazonaws.com/prCommentsSeverityIcons/low_new.svg" alt="LOW" width="24px" align="center">&nbsp;&nbsp;<b style="color:black" >Ensure S3 buckets should have event notifications enabled</b>
            <br>
            &nbsp;&nbsp;&nbsp;&nbsp;Resource: <a href="https://github.com/marynaKK/terragoat/blob/marynaKK-patch-1/main.tf#L113-L141">aws_s3_bucket.logs</a> | Bridgecrew ID: <code>BC_AWS_LOGGING_36</code> | Checkov ID: <a href="https://docs.prismacloud.io/en/enterprise-edition/policy-reference/aws-policies/aws-logging-policies/bc-aws-2-62">CKV2_AWS_62</a>
            <br>
            
</summary>
<h4>How to Fix</h4>

```tf
resource "aws_s3_bucket" "bucket" {
  bucket = "bucket"
}

resource "aws_sns_topic" "bucket_notifications" {
  name = "bucket-notifications"
}

resource "aws_s3_bucket_notification" "bucket_notification" {
  bucket = aws_s3_bucket.bucket.id

  topic {
    topic_arn     = aws_sns_topic.bucket_notifications.arn
    events        = ["s3:ObjectCreated:*"]
    filter_prefix = "logs/"
  }
}
```

<h4>Description</h4>

This policy detects if the S3 bucket event notifications are not enabled in AWS settings.

</details>
<p><img src="https://bc-network-graph-icons.s3.us-west-2.amazonaws.com/prCommentsBotLogos/bc.svg" alt="Bridgecrew" width="18px">&nbsp;<small>Comment by Bridgecrew Bot</small></p>